pkg/binder/binding/resourcereservation/resource_reservation.go

```diff
@@ -36,7 +36,6 @@ type Interface interface {
 const (
 	resourceReservation            = "resource-reservation"
 	gpuReservationPodPrefix        = "gpu-reservation"
-	scalingPodsNamespace           = "runai-scale-adjust"
 	gpuIndexAnnotationName         = "run.ai/reserve_for_gpu_index"
```

Comment:
I assume this can't be changed yet?

Reply:
The new default is `kai-scale-adjust` but it is also configurable, is that what you asked about?

Reply:
About `gpuIndexAnnotationName` - not yet